src/main/java/entities/Player.java

```diff
@@ -6,6 +6,7 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Iterator;
```

Comment:
We can remove this now since I just commented on how we can work around using an iterator.
```suggestion
```